tests/unit/extensions/test_kde_neon_6.py

```diff
@@ -221,12 +291,8 @@ def assert_get_part_snippet(kde_neon_6_instance):
                 {
                     "LD_LIBRARY_PATH": (
                         "/snap/kde-qt6-core22-sdk/current/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}:"
-                        "/snap/kde-qt6-core22-sdk/current/usr/lib:"
                         "/snap/kf6-core22-sdk/current/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}:"
-                        "/snap/kf6-core22-sdk/current/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}"
```

Comment:
Aside from this being a very very welcomed PR :tada:, is there a reason to remove these?

Reply:
Well gosh, something went terribly wrong. I don't have these changes in my local branch. Did I somehow miss a merge? How would it let me.